app/views/layouts/_sidebar.html.slim

```diff
@@ -1,3 +1,4 @@
-div.sidebar.sidebar-left.sidebar-animate#site-navigation-sidebar class=sidebar_classes
-  ul.nav.navbar-stacked
-    = yield
+div class=sidebar_classes
+  nav.navbar-default.sidebar role='navigation'
+    div.collapse.navbar-collapse#site-navigation-sidebar
```

Comment:
Okay, I'm a bit rusty on this part, got it.

sidebar creates a new sidebar. sidebar_items actually creates the items inside the sidebar...